src/azurelargeinstance/azext_azurelargeinstance/tests/latest/test_azurelargeinstance.py

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import *
+
+
+class AzurelargeinstanceScenario(ScenarioTest):
```

Comment:
please also add live tests on `list`, `show`, `update` and other added cmds

Reply:
@AllyW For the tests since there's no arm put method, we'll need to use our own subscription and server for operation commands such as restarting a large instance. We won't be able to provide a server to run the operation commands. Utilizing our existing resources will conflict with our existing test runners such as api test runners. 
May I get an exception for the tests?

Reply:
Acknowledged